lib/matplotlib/tests/test_mathtext.py

```diff
@@ -109,6 +109,8 @@
     r'$,$ $.$ $1{,}234{, }567{ , }890$ and $1,234,567,890$',  # github issue 5799
     r'$\left(X\right)_{a}^{b}$',  # github issue 7615
     r'$\dfrac{\$100.00}{y}$',  # github issue #1888
+    r'$x \overset{f}{\rightarrow} y$',  # github issue #18241
+    r'$x \underset{f}{\rightarrow} y$',  # github issue #18241
```

Comment:
Please consider merging these two tests together into a single test case, as this should result in a nearly twofold save in file size for the vector formats (which won't have to encode the glyphs twice).
The commits should be squashed after merging the test cases to remove the separate files from the git history.

Reply:
That makes sense, so after merging it would look like:
`r'$x \overset{f}{\rightarrow} \underset{f}{\rightarrow} y$'`

Reply:
thanks